data/HOGE/data.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "hoge.finance",
+  "symbol": "HOGE",
+  "decimals": 9,
+  "description": "Hoge Finance ($HOGE) is a deflationary ERC720 token featuring a 2% burn and 1% reflection mechanism on every transaction.\n\nOriginally deployed on the Ethereum L1 network.",
+  "website": "www.Hoge.report",
```

Comment:
@blasthoge 

Please update this as per @cfluke-cb 's request and also the other links to match 🙏

Reply:
on it now